.github/CODEOWNERS

```
@@ -1 +1,2 @@
 * @codacy/support
+/docs/docs/release-notes/self-hosted/ @codacy/support @codacy/qa
```

I think the `docs/` directory is repeated (judging by [these examples](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax)):

```suggestion
/docs/release-notes/self-hosted/ @codacy/support @codacy/qa
```